utils/scripts/ocm/ocm.py

```diff
@@ -863,6 +873,29 @@ def wait_for_osd_cluster_to_get_deleted(self, timeout=3600):
             required=True)
         uninstall_rhods_parser.set_defaults(func=ocm_obj.uninstall_rhods_addon)
 
+        # Argument parsers for install_rhoda_addon
+        install_rhoda_parser = subparsers.add_parser(
+            'install_rhoda_addon',
+            help=("Install rhoda addon cluster."),
+            formatter_class=argparse.ArgumentDefaultsHelpFormatter)
+        required_install_rhoda_parser = install_rhoda_parser.add_argument_group('required arguments')
+        required_install_rhoda_parser.add_argument("--cluster-name",
```

Comment:
Not very important... it would be good to have a consistent visualizer. I believe it would be good to run `black` a formatter on this file.

More information on this formatting tools is available at https://black.readthedocs.io/en/stable/

Reply:
Formatting of python and robot files are included in the upcoming PR. We are planning to include PEP8 (python linter), Black and Robot tidy for formatting